hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -159,6 +165,12 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 		if err := validateAutoscalingParameters(nodePool); err != nil {
 			// TODO (alberto): set status conditions:
 			// type: AutoscalingEnabled, status: true/false, reason: asExpected/validationFailed
+			meta.SetStatusCondition(&nodePool.Status.Conditions, metav1.Condition{
+				Type:    hyperv1.AutoscalingEnabledConditionType,
+				Status:  metav1.ConditionFalse,
+				Reason:  "validationFailed",
```

Comment:
pretty sure this should be "ValidationFailed" for consistency with the rest of the system